applications/RomApplication/python_scripts/ConvectionDiffusionROMsolver.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+
+# Import base class file
+from KratosMultiphysics.ConvectionDiffusionApplication import convection_diffusion_base_solver
+import KratosMultiphysics.RomApplication as romapp
+
+def CreateSolver(main_model_part, custom_settings):
+    return ROMSolver(main_model_part, custom_settings)
+
+class ROMSolver(convection_diffusion_base_solver.ConvectionDiffusionBaseSolver):
+    """The stationary class for ROM convection-diffusion solvers.
+
+    Public member variables:
+    stationary_settings -- settings for the implicit dynamic solvers.
+
+    See convection_diffusion_base_solver.py for more information.
+    """
+
+    def __init__(self, main_model_part, custom_settings):
+        # Set defaults and validate custom settings.
+        self.stationary_settings = KratosMultiphysics.Parameters(r"""{}""")
+
+        # Construct the base solver and validate the remaining settings in the base class
+        super(ROMSolver, self).__init__(main_model_part, custom_settings)
+
+        # Overwrite the base solver minimum buffer size
```

Comment:
Why are you overwriting the base solver buffer size? Are you only solving steady problems? If so it is, I'd specify this in the comment as a reminder to remove this in the future.

Reply:
I think in this case makes sense to derive from the steady solver. However, as soon as we move to transient problems, this would imply to create a new solver rather than reusing this one. 

Long story short, the question in here is how we integrate the ROM in the current developments. I think we need to start this discussion as soon as possible.